### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by cocoahero**
Updates the README with new documentation for managing carts and checkout.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,102 +8,23 @@
...
-    - [Customer Orders](#customer-orders-)
-    - [Customer Update](#customer-update-)
-
-- [Sample application](#sample-application-)
-- [Contributions](#contributions-)
-- [Help](#help-)
-- [License](#license-)
-
-### Documentation [⤴](#table-of-contents)
+The Mobile Buy SDK makes it easy to create custom storefronts in your mobile app. The SDK connects to the Shopify platform using the Storefront GraphQL API, and supports a wide range of native storefront experiences.
```
Comment (markmur) [inline README.md:11]:
```suggestion
The Mobile Buy SDK makes it easy to create custom storefronts in your mobile app. The SDK connects to the Shopify platform using the Storefront GraphQL API and supports a wide range of native storefront experiences.
```

File: README.md [hunk 2]
```diff
@@ -8,102 +8,23 @@
...
-
-- [Search](#search-)
-
-  - [Fuzzy matching](#fuzzy-matching-)
-  - [Field matching](#field-matching-)
-  - [Negating field matching](#negating-field-matching-)
-  - [Boolean operators](#boolean-operators-)
-  - [Comparison operators](#comparison-operators-)
-  - [Exists operator](#exists-operator-)
-
```
[the remark this answers is not in the document]
Comment (markmur) [reply README.md:46]: Oh nice!

File: README.md [hunk 3]
```diff
@@ -243,33 +164,33 @@ let name = shop.field("name") as! String
...
 let query = Storefront.buildQuery { $0
     .node(id: id) { $0
-        .onOrder { $0
+        .onProduct { $0
             .id()
-            .createdAt()
+            .title()
         }
     }
 }
```
[the remark this answers is not in the document]
Comment (cocoahero) [reply README.md:182]: The `$0` is what makes the code valid. It's really `$0.onProduct` but for the sake of readability and mimicking GraphQL queries, the $0 is formatted at the top.

**Review by markmur — APPROVED** [review]

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (markmur) [reply README.md:182]: Ah, good to know

File: README.md [hunk 4]
```diff
@@ -821,633 +462,270 @@ query {
...
-    lineItems: .value([
-        Storefront.CheckoutLineItemInput.create(variantId: GraphQL.ID(rawValue: "mFyaWFu"), quantity: 5),
-        Storefront.CheckoutLineItemInput.create(variantId: GraphQL.ID(rawValue: "8vc2hGl"), quantity: 3),
-    ])
-)
-```
+#### Creating a Cart
 
-The checkout input object accepts other arguments like `email` and `shippingAddress`. In our example we don't have access to that information from the customer until a later time, so we won't include them in this mutation. Given the checkout input, we can execute the `checkoutCreate` mutation:
+To create a `Cart`, you will use the `cartCreate` mutation. If you do not specify any input arguments, you will simply get an empty `Cart`.
```
Comment (igrigorik) [inline README.md:608]: Let's call out that while this is possible, it's not desired. You shouldn't pre-emptively create carts: call cartCreate when you have something to put into the cart.

**Review by igrigorik — APPROVED** [review]
ship it!

**Pull request merged**